pkg/controller/workload/cache/workload_cache.go

```diff
@@ -28,7 +28,7 @@ import (
 type WorkloadCache interface {
 	GetWorkloadByUid(uid string) *workloadapi.Workload
 	GetWorkloadByAddr(networkAddress NetworkAddress) *workloadapi.Workload
-	AddWorkload(workload *workloadapi.Workload)
+	AddWorkload(workload *workloadapi.Workload) ([]string, []string)
```

Comment:
```suggestion
	AddOrUpdateWorkload(workload *workloadapi.Workload) ([]string, []string)
```

And add comments on the return values